contents.md

```diff
@@ -0,0 +1,56 @@
+[![MIT Licensed][icon-mit]][license]
+[![Ideas and useful links][icon-ideas]][ideas]
+[![Awesome][icon-awesome]][awesome]
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+[![Gitter][icon-chat]][chat]
+
+# Front-End Course. Contents
+
+## Stage 0. Admission
+
+The online stage of _Front-End Course_. Following tasks should be completed
+at home to participate offline stage. Learn more in [README](README.md).
+
+### General Basics
+ 0. [Git Basics](tasks/git-intro.md)
+ 1. [Linux CLI and Networking](tasks/linux-cli-http.md)
+ 2. [VCS (hello gitty), GitHub and Collaboration](tasks/git-collaboration.md)
+
+### Front-End Basics
+ 3. [Intro to HTML&nbsp;&&nbsp;CSS](tasks/html-css-intro.md)
```

Comment:
does this need `&nbsp;` in here? Might be me just being grumpy, but strange to put non breaking space as entity and `&amp;` as `&` 🤔 No biggie, can merge as is.

Reply:
Thank you for suggestion. Just forgot about it